CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 5.1.0 (2021-10-28)
+
+- Added the lookup column type. [#157](https://github.com/blackbaud/skyux-ag-grid/pull/157)
```

Comment:
Should `column` be `cell` like in the PR title?

Reply:
Updated.